raml/sfdc_api.raml

```diff
@@ -0,0 +1,52 @@
+# RAML 0.0
+title:
+version: v1
+baseUrl: http://example.com/api
+/alert:
+  post:
+    description: Create or update Case in SFDC
+    responses:
+      200:
+    body:
+      application/json:
+        schema |
+          {"$schema": “http://json-schema.org/schema”,
+            "type":"object"
+            "description": "Alert Data"
+            "properties": {
+              "description":   {"type": "json"},
+              "alertId":       {"type": "string"},
+              "subject":       {"type": "string"},
+              "environment":   {"type": "string"},
+              "alertPriority": {"type": "string"},
+              "alertHost":     {"type": "string"},
```

Comment:
I saw that many Host can be affected by one alert, is it true? if it is true this should be a list